code/game/objects/items/weapons/storage/business_card.dm

```diff
@@ -33,9 +33,33 @@
 /obj/item/storage/business_card_holder/plastic
 	icon_state = "holder_plastic"
 
+// Business cards (copied from lunchbox code)
+var/list/business_cards_ = list(
+	/obj/item/paper/business_card,
+	/obj/item/paper/business_card/alt,
+	/obj/item/paper/business_card/rounded,
+	/obj/item/paper/business_card/glass,
+	/obj/item/paper/business_card/glass/b,
+	/obj/item/paper/business_card/glass/g,
+	/obj/item/paper/business_card/glass/s,
+	/obj/item/paper/business_card/glass/w
+	)
+
+/proc/business_cards()
+	if(!(business_cards_[business_cards_[1]]))
+		business_cards_ = init_cardable_list(business_cards_)
+	return business_cards_
+
+/proc/init_cardable_list(var/list/cardables)
+	. = list()
+	for(var/card in cardables)
+		var/obj/O = card
+		.[initial(O.name)] = card
```

Comment:
This is all unnecessary and can just be replaced by a `typesof(/obj/item/paper/business_card)` loop if you ever need it.